docs/examples.rst

```diff
@@ -991,8 +991,12 @@ The script can be run with: ``python ./eventscanner.py <your JSON-RPC API URL>``
                     # Get UTC time when this event happened (block mined timestamp)
                     # from our in-memory cache
                     block_when = get_block_when(block_number)
-
-                    logger.debug(f"Processing event {evt["event"]}, block: {evt["blockNumber"]} count: {evt["blockNumber"]}")
```

Comment:
I also noticed that `count` and `block` both refer to `evt["blockNumber"]`. I'm not yet sure if this was intentional. If this is not intended, we can address this in the same pull request.

Reply:
I don't think that was intentional, please feel free to update! Thanks!